### ISSUE TITLE
Psremote play 6.0 incompatibility

### ISSUE THREAD

**Issue by Hevendor** [thiagomiranda3/PlaystationMacro#1]
As seen you have added the issue option in here all write here, too.
The new version of psremote play doesn't work with ps macro, when you open your program the inputs are not recognize anymore, and they start working again after closing it.
I've tried with an older version of psremote play and psmacro still works on there, but you're limited to a local network connectivity and it doesn't work well with frequent disconnections.

**Comment by thiagomiranda3** [comment]
Hello. I will look into it as soon as I have free time. For now, someone said me that the older PS Remote still works as expected.
Try keeping using the older version of PS Play for now.

As soon as I can get this fixed, I generate a new release version

**Comment by DiggsAsura** [comment]
I only downloaded the PS Remote Play now - so obviously have the new version. Is there any way to get hold of an older version?

**Comment by honkuen** [comment]
> Hello. I will look into it as soon as I have free time. For now, someone said me that the older PS Remote still works as expected. Try keeping using the older version of PS Play for now.
> 
> As soon as I can get this fixed, I generate a new release version

As long as I updated the PS4 firmware to 10.50, the PlayStationMacro won't work even I downgraded the PS Remote Play to 5.5.0.

**Comment by Hevendor** [comment]
With version 5.5, when it asks to update, deny it, you will then connect locally. Everytime you'll need to use a new pairing code from the console's settings.

**Comment by DiggsAsura** [comment]
It won't find my PS5 whis way for some reason, so when I deny -> register
pin -> won't find PS5. Even they are on same network and works fine in
6.0.0.

This is on Windows 11 on a Mac through Parallels though, maybe they screw
up with the network settings.


Mvh
Kenneth Bæver Bjerke
+47 966 21 811

**Comment by Hevendor** [comment]
Probably a network problem. Try to DMZ the ps5 or something like that, it worked when my ps4 had network problems.

**Comment by akaymd** [comment]
It is possible that RemotePlay is detecting API Hooks to prevent macros.
I tried [EasyHook Tutorial](http://easyhook.github.io/tutorials/remotefilemonitor.html) and got the same behavior.
When the Hook is injected, the controller becomes unresponsive.
The controller regains response by removing the Hook.

**Comment by thiagomiranda3** [comment]
Yes. This weekend I was looking into the problem. They seem to have some antihook mechanism applied in this version.

Even if your hook simply return the original method, we still get the same problem.

Later I will try making the hook be injected before the PS Remote run by using the CreateAndInject method that Easy hook has. Hope this works 😂

**Comment by akaymd** [comment]
Excellent!

EasyHook Tutolial's CreateAndInject worked.
I hope there are no other obstacles.

**Comment by thiagomiranda3** [comment]
You are the boss man, don't you want to fix this issue and submit a PR for me? You would be one of the contributors of the project

**Comment by akaymd** [comment]
Okay, I will try... but it doesn't work.

I applied patch for CreateAndInject to PS4RemotePlayInterceptor.
The RemotePlay process starts, but dies before the window appears.
On EasyHook Tutorial, works properly.

Do you have any ideas?

**Comment by akaymd** [comment]
I tried to push above changes, but I don't seem to have write permission to the repository.
Can I get permission?

I have never pushed to someone else's repository on github and am not sure.

**Comment by thiagomiranda3** [comment]
Oh.
The first thing you need to do is to fork the project. This will create a copy of them in your own repository.

Them you add you push your change their and open a pull request, sending the change of your repository to mine.

I will review it and accept later

**Comment by akaymd** [comment]
I'll try.
Thanks.

**Comment by realshadow** [comment]
Thanks for the great work!

In the meantime, if anyone stumbles on this issue I managed to make it work by downloading older version of remote play (5.5) and using remote play version patcher (you can find it on github).

**Comment by thiagomiranda3** [comment]
@akaymd , thanks for the PR. I will review it as soon as I have some free time. Probably tomorrow and at most on this weekend.

**Comment by hieurick** [comment]
any update bro ?

**Comment by thiagomiranda3** [comment]
Not yet.
The new PS Remote Play has an anti hook protection that makes this program useless. The only workaround so far to continue using it is by downloading an older version of PSRemotePlay:

https://ps5-remote-play.uptodown.com/windows/descargar/81670958

Download it and when it ask for update to new version, click No. You will have to link it to your PS4/PS5 everytime, but at least the macro will work on this version.

**Comment by hieurick** [comment]
Thanks bro. 
But I get this error when using your version with simple script:
![image](https://user-images.githubusercontent.com/85996144/229208542-11bff2a5-cfcf-4d50-8290-9c1fc5af5631.png)

I tried using Komefai's Version but it didn't work:
![image](https://user-images.githubusercontent.com/85996144/229209117-bb04707f-2968-40bf-a279-3c9fe051d74e.png)

Please help !
Many thanks !

**Comment by martinjenik** [comment]
Hi guys, does anyone know where to download PSRemote in version5.5? Could not find any safe one for download...Thx
